VOKMockUrlProtocol.m

```diff
@@ -111,6 +118,28 @@ - (NSArray *)resourceNames
         [resourceName appendFormat:queryFormat, self.request.URL.query];
     }
     
+    if (headersToEncode) {
```

Comment:
Even if there are no headers to encode, we still need to add a separator to distinguish the header segment of the file name.